locales/es.json

```diff
@@ -36,5 +36,5 @@
   "Type to search": "Escribe para buscar",
   "No results for your search, try something different.": "No hay resultados para tu búsqueda, prueba algo diferente.",
   "JavaScript license information": "Información de licencia JavaScript",
-  "Toggle dark mode": "Cambiar modo oscuro"
+  "Toggle dark mode": "Cambiar a modo oscuro"
```

Comment:
Maybe something like "Alternar modo oscuro" would be better? 🤔 because that button also allows you to change from dark to light mode

Reply:
Modification done. Your point of view go to the point. Kudos!